src/test/java/com/linkedin/kafka/clients/auditing/abstractimpl/ConfigureAuditorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.linkedin.kafka.clients.auditing.abstractimpl;
+
+import com.linkedin.kafka.clients.auditing.NoOpAuditor;
+import com.linkedin.kafka.clients.consumer.LiKafkaConsumer;
+import com.linkedin.kafka.clients.consumer.LiKafkaConsumerImpl;
+import com.linkedin.kafka.clients.producer.LiKafkaProducer;
+import com.linkedin.kafka.clients.producer.LiKafkaProducerConfig;
+import com.linkedin.kafka.clients.utils.tests.AbstractKafkaClientsIntegrationTestHarness;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+import java.util.Properties;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+public class ConfigureAuditorTest extends AbstractKafkaClientsIntegrationTestHarness {
+  @BeforeMethod
+  @Override
+  public void setUp() {
```

Comment:
These `setUp` and `tearDown` methods are actually quite expensive. It would be good to see if we can create producer and consumer without bring up the servers.